new_contributor_guide.md

```diff
@@ -0,0 +1,21 @@
+## Guide for New Contributors
+
+Hello, and welcome to the Rust-ML group! We're glad you may be interested in contributing to our efforts. First, please take the time to review our main README document, particularly around the "What do we do" section. 
+
+In general, the Rust-ML group emphasizes the Bazaar, not the Cathedral. In the [Zulip chat](https://rust-ml.zulipchat.com/), we regularly see people who say "Hi, I'm new here and would love to help contribute!" under the assumption that there may be a comprehensive Grand Plan that they can be directed towards. While well-intentioned, this approach is a little difficult to respond to because, while some individual projects do have roadmaps, the working group is not a Group with a Plan, but instead a collection of individuals working on things they're personally interested in. 
```

Comment:
Yup, I can definitely put a link in there! I also agree that creating a meta-issue like that would be a good idea. I think that would be really helpful for lowering the barrier to entry for participating, since even people with write-access to the Task Board don't often update it, and conversations on Zulip get lost pretty quickly

Reply:
(To be clear, I think part of that is because the tasks themselves don't change too often, so that wasn't an attempt to point fingers or anything. The rate of update of the task board is not a huge issue in my mind)